### PR TITLE
feat(textColor): Adds a11y textColor attribute; small refactor.

### PR DESCRIPTION
**Pull request by aendrew**
Also adds:
- uk.parties: all party data as an Object.
- uk.partiesArray: same as above but as an array, with additional "key" attribute corresponding to above.

Logic for the object keys: it's the party `shortName`, uppercased, with spaces removed.

### CODE REVIEW

File: src/uk/index.js [hunk 1]
```diff
@@ -5,154 +5,196 @@
...
+    color: "#7200ab",
+    textColor: "white",
+    whiteOverlayOpacity: 0.3,
+    shortName: "Ukip",
+    formattedName: "Ukip",
+    adjective: "Ukip"
+  },
+  PC: {
+    color: "#990000",
+    textColor: "black",
```
Comment (joannaskao) [inline src/uk/index.js]: We should use white here. Black doesn't pass any of the accessibility guidelines

File: src/uk/index.js [hunk 2]
```diff
@@ -5,154 +5,196 @@
 
 import removeAccents from "remove-accents";
 
+export const parties = {
+  CON: {
+    color: "#149adb",
+    textColor: "white",
```
Comment (joannaskao) [inline src/uk/index.js]: I think we've used white here in the past, but contrast checkers for accessibility think black text on this blue is better.

File: src/uk/index.js [hunk 3]
```diff
@@ -5,154 +5,196 @@
...
+    whiteOverlayOpacity: 0.2,
+    shortName: "Con",
+    formattedName: "Conservative",
+    adjective: "Conservative party"
+  },
+  LAB: {
+    color: "#cf4d3c",
+    textColor: "white",
```
Comment (joannaskao) [inline src/uk/index.js]: Similarly, we've done white before, but the contrast ratio is greater if we use black text.

File: src/uk/index.js [hunk 4]
```diff
@@ -5,154 +5,196 @@
...
+    color: "#210066",
+    textColor: "white",
+    whiteOverlayOpacity: 0.5,
+    shortName: "DUP",
+    formattedName: "Democratic Unionist party",
+    adjective: "Democratic Unionist party"
+  },
+  UUP: {
+    color: "#3f67cc",
+    textColor: "black",
```
Comment (joannaskao) [inline src/uk/index.js]: Better contrast if we use white here.

File: src/uk/index.js (hunk 1 above)
Comment (aendrew) [reply src/uk/index.js]: :+1:

File: src/uk/index.js (hunk 2 above)
Comment (aendrew) [reply src/uk/index.js]: @cnev177 uses white in the design so I'll stick with that unless told otherwise.

File: src/uk/index.js (hunk 3 above)
Comment (aendrew) [reply src/uk/index.js]: Ditto; will check with @cnev177 next she's in.

File: src/uk/index.js (hunk 4 above)
Comment (aendrew) [reply src/uk/index.js]: :+1:

**Pull request merged**